discovery-provider/alembic/versions/5aaa6198def8_user_primary_secondary_columns.py

```diff
@@ -0,0 +1,30 @@
+"""user primary secondary columns
+
+Revision ID: 5aaa6198def8
+Revises: 6d1b38f242fe
+Create Date: 2020-12-22 13:00:09.925084
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5aaa6198def8'
+down_revision = 'f82eb376f471'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('primary', sa.Integer(), nullable=True))
```

Comment:
is it intentional to leave these col values as nullable?

Reply:
/ is there going to be a retroactive fill for primaries and secondaries for all users?

Reply:
yes these must be nullable for backwards compatibility during the interim where users have yet to upgrade